apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -414,8 +414,8 @@ function AutoAssessmentGroupsModal({
   csrfToken: string;
 }) {
   return Modal({
-    id: 'autoAssessmentGroupsModal',
-    title: 'Auto new group setting',
+    id: 'randomAssessmentGroupsModal',
+    title: 'Random group assignments',
```

Comment:
Nit only because we're touching this code: the inputs in this modal should have `type="number"` and `required`.

If we wanted to be clever, we could pre-populate this based on the min/max configured in the group settings.

I will by no means block on either of these.

Reply:
> Nit only because we're touching this code: the inputs in this modal should have `type="number"` and `required`.

Done in ffbc03f2c634a0ada3667d7ff4dfe59e36443d22. Also added `min="1"`.

> If we wanted to be clever, we could pre-populate this based on the min/max configured in the group settings.

This is already done: https://github.com/PrairieLearn/PrairieLearn/blob/ffbc03f2c634a0ada3667d7ff4dfe59e36443d22/apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts#L76-L80